README.md

```diff
@@ -228,3 +228,139 @@ Reference : https://devcenter.heroku.com/articles/container-registry-and-runtime
 Test it out! Push a small change to main branch, wait for the pipeline to complete,
 then check it's visible on the live site.
 
+Replacing Trello Api and Data model with MongoDB.
+
+MongoDB is an open-source NOSQL database.
+
+Part 1: Configure MongoDB database 
+
+MongoDB Atlas service will be used, https://www.mongodb.com/cloud/atlas.
+Free tier will be used to create a MongoDB cluster. 
+Signup process - https://www.mongodb.com/try
+Username and password will be used as the authentication method.
+The access to the clusters is currently set to access from anywhere.
+Refer the "Connect your application" option. It will provide sample code for connecting your application.
+
+Step 2: Installing Pymongo
+
+Python support for MongoDB in the form of PyMongo. Once installed one should be able to interact with MongoDB from the Python shell.
+Configure via poetry: poetry add pymongo pymongo[srv].
+
+Step 3: Connecting to Atlas
+
+Parameter and where to find it?
+
+USER_NAME:  Created in step 1. A list is visible in the "Database Access" menu under the "Security" heading.
+
+PASSWORD: Created in step 1. If lost you'll have to to change the password in the "Database Access" menu.
+
+MONGO_URL: Visible in the "Connect" menu of your cluster. Look for a URL that ends .mongodb.net
+DEFAULT_DATABASE The name of the default database: you choose what this is.
+
+Once you have these you can run the following lines of code in the Python shell.
+
+The result should be something similar to the last line. If you run into problems check Atlas' troubleshooting Documentation.
+
+>>> import pymongo
+>>> client = pymongo.MongoClient("mongodb+srv://
+<USER_NAME>:<PASSWORD>@<MONGO_URL>/<DEFAULT_DATABASE>?w=majority")
+>>> client.list_database_names()
+['admin', 'local']
+
+Part 2: Replacing Trello
+
+Step 1: Updating the code
+Replace Trello Data Model with MongoDB One to Many relational data model.
+
+Creating a Database :
+>>> client.list_database_names()
+['test_database', 'admin', 'local']
+
+>>> db = client.Board
+
+Using Postman format the json structure as below data model and extract data from Trello.
+
+>>> board = <'Data sourced from Trello'>
+
+>>> db.Board = board
+
+This will create the collection Board in MongoDB.
+
+
+Mongo Data Model:
+
+{
+
+   "name" : "Test Board",
```

Comment:
Do you ever use these board values? We could simplify this data model a lot by just adding each of the three statuses into the collection separately rather than adding them inside a single datapoint representing the entire board (or even further, we could split the cards up into different objects themselves, either in different collections for the different statuses, or in one collection but with the status values set). Managing all of our data as a single object is going to make this more difficult to maintain, and a struggle to keep scalable!